### PR TITLE
Add web server configuration

### PR DESCRIPTION
**Pull request by fritzmg**
This (finally) adds the minimum configuration necessary for Apache and NGINX to the documentation.

### CODE REVIEW

File: docs/manual/installation/system-requirements.de.md [hunk 1]
```diff
@@ -250,10 +250,98 @@ dann definiere diesen entsprechend, um für zukünftige Contao Versionen gerüst
 
 siehe auch: https://symfony.com/doc/current/configuration/override_dir_structure.html#override-the-public-directory
 
+
+### Web-Server Konfiguration
```
Comment (netzarbeiter) [inline docs/manual/installation/system-requirements.de.md]:
```suggestion
### Webserver-Konfiguration
```

File: docs/manual/installation/system-requirements.de.md [hunk 2]
```diff
@@ -250,10 +250,98 @@ dann definiere diesen entsprechend, um für zukünftige Contao Versionen gerüst
...
+
+In der Konfiguration des Web-Servers muss sichergestellt sein, dass alle Anfragen von der Applikation über die `index.php` im öffentlichen
```
Comment (netzarbeiter) [inline docs/manual/installation/system-requirements.de.md]:
```suggestion
In der Konfiguration des Webservers muss sichergestellt sein, dass alle Anfragen von der Applikation über die `index.php` im öffentlichen
```

File: docs/manual/installation/system-requirements.de.md [hunk 3]
```diff
@@ -250,10 +250,98 @@ dann definiere diesen entsprechend, um für zukünftige Contao Versionen gerüst
...
+Verzeichnis verarbeitet werden (via »URL-Rewriting«). Wie diese Konfiguration aussehen musst hängt von der eingesetzten Web-Server Software
```
Comment (netzarbeiter) [inline docs/manual/installation/system-requirements.de.md]:
```suggestion
Verzeichnis verarbeitet werden (via »URL-Rewriting«). Wie diese Konfiguration aussehen musst hängt von der eingesetzten Webserver-Software 
```

File: docs/manual/installation/system-requirements.de.md [hunk 4]
```diff
@@ -250,10 +250,98 @@ dann definiere diesen entsprechend, um für zukünftige Contao Versionen gerüst
...
+In der Konfiguration des Web-Servers muss sichergestellt sein, dass alle Anfragen von der Applikation über die `index.php` im öffentlichen
+Verzeichnis verarbeitet werden (via »URL-Rewriting«). Wie diese Konfiguration aussehen musst hängt von der eingesetzten Web-Server Software 
+ab. Weit verbreitete Beispiele sind Apache und NGINX:
+
+{{< tabs groupId="web-server-config" >}}
+
+{{% tab name="Apache" %}}
+Für Apache stellt Contao eine [Standard `.htaccess`](https://github.com/contao/contao/blob/5.0.7/manager-bundle/skeleton/public/.htaccess) 
+Datei im öffentlichen Verzeichnis zur Verfügung. Damit diese Datei von Apache verarbeitet wird muss sichergestellt sein, dass die Direktive
+`AllowOverride All` für das `Directory` in der `VirtualHost` Definition der Web-Server Konfiguration vorhanden ist. Darüberhinaus muss das
```
Comment (netzarbeiter) [inline docs/manual/installation/system-requirements.de.md]:
```suggestion
`AllowOverride All` für das `Directory` in der `VirtualHost` Definition der Webserver-Konfiguration vorhanden ist. Darüberhinaus muss das
```

File: docs/manual/installation/system-requirements.de.md [hunk 5]
```diff
@@ -250,10 +250,98 @@ dann definiere diesen entsprechend, um für zukünftige Contao Versionen gerüst
...
+Apache Modul `mod_rewrite` aktiv sein, damit URLs wie `https://example.com/contao/install` möglich sind. Falls beides nicht zutrifft würden
```
Comment (netzarbeiter) [inline docs/manual/installation/system-requirements.de.md]:
```suggestion
Apache-Modul `mod_rewrite` aktiv sein, damit URLs wie `https://example.com/contao/install` möglich sind. Falls beides nicht zutrifft würden
```

File: docs/manual/installation/system-requirements.de.md [hunk 6]
```diff
@@ -250,10 +250,98 @@ dann definiere diesen entsprechend, um für zukünftige Contao Versionen gerüst
...
+nur URLs wie `https://example.com/index.php/contao/install` möglich sein.
+
+Für Contao muss auch die Einstellung `Options FollowSymlinks` in der `Directory` Konfiguration aktiv sein, da Symlinks zum Einsatz kommen.
+
+Eine minimale `VirtualHost` Konfiguration für den Apache Web-Server könnte also z. B. so aussehen (`…/public` mit `…/web` austauschen für
```
Comment (netzarbeiter) [inline docs/manual/installation/system-requirements.de.md]:
```suggestion
Eine minimale `VirtualHost` Konfiguration für den Apache-Webserver könnte also z. B. so aussehen (`…/public` mit `…/web` austauschen für
```

File: docs/manual/installation/system-requirements.de.md [hunk 7]
```diff
@@ -250,10 +250,98 @@ dann definiere diesen entsprechend, um für zukünftige Contao Versionen gerüst
...
+        Require all granted
+        Options FollowSymlinks
+    </Directory>
+</VirtualHost>
+```
+
+{{% /tab %}}
+
+{{% tab name="NGINX" %}}
+Am wichtigsten ist es sicherzustellen, dass alle Anfragen die nicht an eine existierende Datei gehen an die PHP Applikation zur Verarbeitung
```
Comment (netzarbeiter) [inline docs/manual/installation/system-requirements.de.md]:
```suggestion
Am wichtigsten ist es sicherzustellen, dass alle Anfragen die nicht an eine existierende Datei gehen an die PHP-Applikation zur Verarbeitung
```

**Review by Toflar — APPROVED** [review]

File: docs/manual/installation/system-requirements.de.md [hunk 8]
```diff
@@ -250,10 +250,98 @@ dann definiere diesen entsprechend, um für zukünftige Contao Versionen gerüst
...
+    location ~ ^/(preview|contao-manager\.phar)\.php(/|$) {
+        # the exact FastCGI configuration depends on your environment
+        fastcgi_pass unix:/var/run/php/php8.1-fpm.sock;
+        fastcgi_split_path_info ^(.+\.php)(/.*)$;
+        include fastcgi.conf;
+    }
+}
+```
+
+Eine vollständige NGINX Konfiguration enthält normalerweise mehr Einträge, bspw. um das "not found logging" für statische Resourcen wie
```
Comment (netzarbeiter) [inline docs/manual/installation/system-requirements.de.md]:
```suggestion
Eine vollständige NGINX Konfiguration enthält normalerweise mehr Einträge, bspw. um das "not found logging" für statische Ressourcen wie
```

**Review by netzarbeiter — CHANGES_REQUESTED** [review]
Thank you, Fritz

File: docs/manual/installation/system-requirements.de.md [hunk 9]
```diff
@@ -250,10 +250,98 @@ dann definiere diesen entsprechend, um für zukünftige Contao Versionen gerüst
 
 siehe auch: https://symfony.com/doc/current/configuration/override_dir_structure.html#override-the-public-directory
 
+
+### Webserver-Konfiguration
+
+In der Konfiguration des Webservers muss sichergestellt sein, dass alle Anfragen von der Applikation über die `index.php` im öffentlichen
+Verzeichnis verarbeitet werden (via »URL-Rewriting«). Wie diese Konfiguration aussehen muss hängt von der eingesetzten Web-Server Software
```
Comment (netzarbeiter) [inline docs/manual/installation/system-requirements.de.md]:
```suggestion
Verzeichnis verarbeitet werden (via »URL-Rewriting«). Wie diese Konfiguration aussehen muss hängt von der eingesetzten Webserver-Software 
```

**Review by netzarbeiter — APPROVED** [review]

**Pull request merged**